priv/elevators/elevator_redundancy_data.json

```diff
@@ -0,0 +1,1057 @@
+[
+  {
+    "id": "814",
+    "station_id": "place-alfcl",
```

Comment:
It appears we don't use this field at all in the current code. Is there a future need for it?

Reply:
I anticipated a need for it, but that may be premature. I assume it will be necessary in [this](https://app.asana.com/0/1185117109217422/1208677840815949/f) task, but so will other fields I didn't include. Can remove now and let future implementation add what is needed.